docs/bdd/basic-usage.rst

```diff
@@ -7,8 +7,12 @@ Now let us show you the flow.
 
 Describing features
 -------------------
-Let's start with writing our feature file, which will contain answers to the most important questions: Why (benefit), who (actor using the feature) and what (the feature itself). It should also include scenarios, which serve as examples of how things supposed to work.
-``features/addressing/managing_countries/adding_country.feature``
+
+Let's start with writing our feature file, which will contain answers to the most important questions:
+Why (benefit), who (actor using the feature) and what (the feature(benefit) itself).
```

Comment:
Missing space before `(`

Reply:
Shouldn't the "why" part be a reason instead of benefit?

Reply:
Benefit is the reason :D